SourceX/controls/plrctrls.cpp

```diff
@@ -800,26 +800,26 @@ bool IsPathBlocked(int x, int y, int dir)
 	return !PosOkPlayer(myplr, d1x, d1y) && !PosOkPlayer(myplr, d2x, d2y);
 }
 
-void WalkInDir(AxisDirection dir)
+void WalkInDir(int playerNumber, AxisDirection dir)

```

Comment:
i thought we agreed to call it playerId :)

Reply:
Oops, forgot about it. Will fix today